Essentials/src/com/earth2me/essentials/commands/Commandkitreset.java

```diff
@@ -0,0 +1,66 @@
+package com.earth2me.essentials.commands;
+
+import com.earth2me.essentials.CommandSource;
+import com.earth2me.essentials.Kit;
+import com.earth2me.essentials.User;
+import org.bukkit.Server;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static com.earth2me.essentials.I18n.tl;
+
+public class Commandkitreset extends EssentialsCommand {
+    public Commandkitreset() {
+        super ("kitreset");
+    }
+
+    @Override
+    public void run(final Server server, final User user, final String commandLabel, final String[] args) throws Exception {
+        if (user.isAuthorized("essentials.kit.reset")) {
+            final User userTo = getPlayer(server, user, args, 0);
```

Comment:
`EssentialsLoopCommand` exists for the sole purpose of commands that act on other targets - you should be extending this class.

Reply:
In addition, the player parameter should come before the kit in the command's syntax - this is consistent with other EssentialsX loop commands.